book/installation.rst

```diff
@@ -98,6 +98,17 @@ number as the second argument of the ``new`` command:
     # Windows
     c:\projects\> php symfony.phar new my_project_name 2.3.23
 
+If you want your project to be based on the last Symfony LTS version, pass
```

Comment:
I understand your point of view
Can you confirm that https://github.com/symfony/symfony-installer/blob/master/src/Symfony/Installer/NewCommand.php#L223 will download automatically the 2.7 when it will be published, even if the 2.3 will still be maintained?
If yes, ```latest``` is the best option imo